doc/source/user_guide/numpy_images.rst

```diff
@@ -33,14 +33,14 @@ types and how ``scikit-image`` treats them.
 NumPy indexing
 --------------
 
-NumPy indexing can be used both for looking at the pixel values and to
-modify them::
+NumPy indexing (0-based!) can be used both for looking at the pixel values
+and to modify them::
 
     >>> # Get the value of the pixel at the 10th row and 20th column
-    >>> camera[10, 20]
+    >>> camera[9, 19]
```

Comment:
I'm a little torn on this nomenclature. Does 10th row mean index 10 or index 9? Does 0th row make sense?

Reply:
@hmaarrfk it does. At least, to me.
I'd rather try to reformulate the comments than to use ...9 indexing.

Reply:
Maybe the problem is that I seldom encounter code that used such a specific index. It is always: “the next index” or “the previous index”

Reply:
@jni Mastermind! I love your suggestion!